distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1008,11 +1009,17 @@ type Table
     @on Widget_Helpers.make_join_condition_selector
     join : Table -> Join_Kind -> Vector (Join_Condition | Text) | Text -> Text -> Problem_Behavior -> Table
     join self right join_kind=Join_Kind.Left_Outer on=(default_join_condition self join_kind) right_prefix="Right " on_problems=Report_Warning =
+        self.join_or_cross_join right join_kind on right_prefix on_problems
+
+    ## PRIVATE
+       Implementation of both `join` and `cross_join`.
+    join_or_cross_join : Table -> Join_Kind | Join_Kind_Cross -> Vector (Join_Condition | Text) | Text -> Text -> Problem_Behavior -> Table
+    join_or_cross_join self right join_kind on right_prefix on_problems =
```

Comment:
I'm wondering if we should move this to a separate file (in `Internal`), that seems to be the direction we are leaning towards recently.

Reply:
I think the fact that this logic had to be factored out is unrelated to code organization, and perhaps it should be kept near the cross_join logic.

Reply:
Can be kept for now, I was just thinking of moving in the direction of keeping helpers in the `Internal` module as is the tendency for recent changes. But there are no 'official' guidelines yet, so I'm ok with either way.